docker/alpine-3.17-amd64-openssl-3.0.x/Dockerfile

```diff
@@ -6,11 +6,11 @@ WORKDIR /usr/src/app
 COPY --from=app . ./
 COPY --from=utils . ./
 
-RUN apk update && apk add openssl1.1-compat
+RUN apk update
 
-# Both OpenSSL 1.1 and OpenSSL 3 must be present after installing openssl1.1-compat
-RUN if [ ! -e /lib/libssl.so.1.1 ]; then echo 'OpenSSL 1.1 not found in /lib/libssl.so.1.1'; exit 1; fi
+# Only OpenSSL 3 is expected to be on this system
```

Comment:
.. as we've removed `openssl1.1-compat`, which instead still exists in the new `alpine-3.17-amd64-openssl-3.0.x` test

Reply:
I guess you meant `alpine-3.17-amd64-openssl-1.1.x`?

Reply:
I did, updated the comment.